packages/blaze/view.js

```diff
@@ -352,13 +352,16 @@ Blaze._materializeView = function (view, parentView, _workStack, _intoArray) {
       view._isInRender = false;
 
       if (! c.firstRun) {
+        // added by ramezrafla@zegenie to skip affecting DOM for unchanged content
+        if (Blaze._deepCompare(lastHtmljs,htmljs)) {
```

Comment:
So currently I worry that this prevents only the innermost HTMLJS trees from being rerendered.